lib/wrap-guide-element.coffee

```diff
@@ -78,6 +79,7 @@ class WrapGuideElement extends HTMLDivElement
     column = @getGuideColumn(@editor.getPath(), @editor.getGrammar().scopeName)
     if column > 0 and @isEnabled()
       columnWidth = @editorElement.getDefaultCharacterWidth() * column
+      columnWidth -= @editor.getScrollLeft()
       @style.left = "#{columnWidth}px"
```

Comment:
@nathansobo: I am think we should use `translate3d` here, so that we don't pay for an extra `Layout` when scrolling left. Performance-wise this is not a concern for now, however: everything is pretty smooth nevertheless.